src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -535,5 +539,20 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
   .v-chip {
     opacity: 1 !important;
   }
+
+  // align badge icon with text
+  .badges .v-icon{
+    margin-top: 4px
+  }
+}
+
+// adjust error container padding for last section
+.people-roles-content.last-section {
```

Comment:
Could you use [last-of-type](https://www.w3schools.com/cssref/sel_last-of-type.asp) to set this padding without needing a conditional classname?

Reply:
Thanks Severin. This last-of-type style is really good.